### PR TITLE
MileStone 1: Mobile version Setup

### PR DESCRIPTION
**Pull request by alvinlouis29**
##I have created this branch as a milestone of my Personal Portfolio website, and The goal is to make it mobile responsive.

##In this milestone, I have used the flex box to align my items on the website to match the UI/UX.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                        hesitate to contact me.
+                    </p>
+                </div>
+                <div class="social">
+                    <ul>
+                        <li ><a href=""><i class="fa fa-twitter"></i></a></li>
+                        <li ><a href=""><i class="fa fa-github"></i></a></li>
+                        <li ><a href=""><i class="fa fa-instagram"></i></a></li>
+                        <li ><a href=""><i class="fa fa-facebook"></i></a></li>
+                    </ul>
```
Comment (Shaher-11) [inline index.html]: Add the missing social icons
right

File: index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+        <section class="hero">
+            <section class="headline">
+                <div class="place primary">
+                    <h2> Hey there. I’m Alvin <br><span class="green"> I’m a software developer </span></h2>
+                </div>
```
Comment (Shaher-11) [inline index.html:36]: Reduce the white space on the top and add some pace between the name and the title
right

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @alvinlouis29,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comments below:point_down:

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: index.html (hunk 2 above)
Comment (rloterh) [reply index.html:36]: This issue was well resolved ✔️

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html]: This issue was well resolved ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Alvin, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 👍.
Congratulations! 🥂


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**